sql/ddl_api.sql

```diff
@@ -64,27 +64,17 @@ CREATE OR REPLACE FUNCTION  set_number_partitions(
     dimension_name          NAME = NULL
 ) RETURNS VOID AS '@MODULE_PATHNAME@', 'ts_dimension_set_num_slices' LANGUAGE C VOLATILE;
 
--- Drop chunks that are older than a timestamp or newer than a timestamp.
+-- Drop chunks older than the given timestamp. If a hypertable name is given,
+-- drop only chunks associated with this table. Any of the first three arguments
+-- can be NULL meaning "all values".
 CREATE OR REPLACE FUNCTION drop_chunks(
-    older_than ANYELEMENT = NULL,
+    older_than "any" = NULL,
```

Comment:
why did we switch to "any" here. Do we want to support different types for older_than and newer_than?